code/modules/cargo/centcom_podlauncher.dm

```diff
@@ -531,16 +531,18 @@ force_open = FALSE, datum/tgui/master_ui = null, datum/ui_state/state = GLOB.adm
 	//UpdateSelector() is here (instead if the if(1) switch block) because it also moves the selector to nullspace (to hide it) if needed
 
 /datum/centcom_podlauncher/proc/launch(turf/A) //Game time started
-	if (isnull(A)) 
+	if (isnull(A))
 		return
 	var/obj/structure/closet/supplypod/centcompod/toLaunch = DuplicateObject(temp_pod) //Duplicate the temp_pod (which we have been varediting or configuring with the UI) and store the result
 	toLaunch.bay = bay //Bay is currently a nonstatic expression, so it cant go into toLaunch using DuplicateObject
 	toLaunch.update_icon()//we update_icon() here so that the door doesnt "flicker on" right after it lands
+	var/shippingLane =  locate(/area/centcom/supplypod/flyMeToTheMoon) in GLOB.sortedAreas
```

Comment:
We have a nice lookup table for these now:
```suggestion
	var/shippingLane = GLOB.areas_by_type[/area/centcom/supplypod/flyMeToTheMoon]
```
It might make sense to use a landmark as well, rather than `forceMove`ing into an area (which I'm surprised works).

Reply:
Eh, i decided against a landmark because theres a grid of places the supplypod can appear in (so that multiple pods dont all overlap on eachother mid-flight)

Reply:
moving an atom directly into an area typically puts it in the lower-leftmost turf rather than a random one